src/screens/OtherProfile/index.js

```diff
@@ -116,16 +117,13 @@ const BioAndChat = (props) => {
           }
         />
       </TouchableOpacity>
-      <TouchableOpacity onPress={toggleSwitch} style={styles.toggleSwitchContainer}>
-        <ToggleSwitch
-          value={isAnonimity}
-          onValueChange={toggleSwitch}
-          labelLeft={
-            isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
-          }
-          styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
-        />
-      </TouchableOpacity>
+      <ToggleSwitchAnon
+        value={isAnonimity}
+        onValueChange={toggleSwitch}
+        labelLeft={
+          isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
+        }
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `TouchableOpacity` with `ToggleSwitchAnon` seems to be a good change as it encapsulates the toggle switch functionality into a separate component. This improves the modularity of the code. However, make sure that the `ToggleSwitchAnon` component handles all the necessary states and events that were previously handled by `TouchableOpacity`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->